frontend/src/Frontend/UI/Dialogs/Receive.hs

```diff
@@ -205,13 +208,18 @@ uiReceiveModal0 model account onClose = Workflow $ do
   let
     done = gate (current showingAddr) doneNext
     deploy = gate (not <$> current showingAddr) doneNext
-
-    infos = (liftA2 . liftA2) (,) netInfo transferInfo
+    ap2' = (liftA2 . liftA2) (&)
```

Comment:
This is a lot less readable than the old implementation imo

Reply:
I don't see how to keep the old implementation without lots of tupling and untupling and non-binary uncurrying.

Reply:
`push` and `sample` is probably the best bet here

Reply:
much clearer